nsapass

```diff
@@ -318,9 +335,7 @@ def nsa_mod(name, index, new_name, new_pass, new_note):
         new_pass = d[section][name][i][KEY_PASS]
     if new_note is None:
         new_note = d[section][name][i][KEY_NOTE]
-    if len(new_note) == 0:
-        new_note = None
```

Comment:
since notes may be absent, some way of clearing notes makes sense. so maybe
* change the check to `new_note != ''` which neatly covers the `None` case as well, or
* provide a flag for clearing the note?

Reply:
oo.  right.  1st option seems simpler to me. will go with 1st.  this way `-n ""` would delete the note.